.github/workflows/test.yml

```
@@ -11,7 +11,26 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.9"]
+        python-version: ["pypy3", "3.6", "3.7", "3.8", "3.9"]
```

I think `pypy3` is an alias for the old `pypy-3.6`

```suggestion
        python-version: ["pypy-3.6", "pypy-3.7", "3.6", "3.7", "3.8", "3.9"]
```

---

yep: https://github.com/pytest-dev/pytest-cov/runs/3660088990?check_suite_focus=true#step:3:7

---

hmm it arrived from pytest-cover here https://github.com/pytest-dev/pytest-cov/pull/58/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R6